rfcs/036-holdings-records/README.md

```diff
@@ -0,0 +1,261 @@
+    It's useful to know that these two volumes form a complete set, but it wouldn't make sense to store this information on the individual items.
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Missing volume 3*.
+    Here the holdings record is even more important, but there's no way to encode the absence of an item.
+
+-   A run of journals, with each issue itemised, and a holdings record titled *100 issues, 1957–1972*.
+    The holdings record can provide useful summary information that isn't easily observed from the items.
+
+### Creating a separate Holdings type
+
+Rather than reuse the `Item` type, we will create a new `Holdings` type with the same fields (at least, initially).
+This will appear as a list on a Work called `"holdings"`.
+
+We will populate it as follows:
+
+*   The `title` will come from the concatenated contents of MARC field 866 subfields $a and $z.
+    We will append the label from 853 and 863 (newline-separated), one per 853/863 pair.
+    (We might need to split this out if it becomes unwieldy.)
+
+*   The `locations` list will contain:
+
+    *   A digital location using field 856 if the location in fixed field 40 is `elro` (online)
+    *   A physical location using fixed field 40 for the location type/label, and 949 for the shelfmark.
+
+*   For now, we won't add any identifiers.
+    We could add them as h-prefixed Sierra IDs, but I don't see much use of those identifiers -- usually people use the b number when discussing the associated holdings record.
+
+### Possible future enhancements
+
+Although we're treating items and holdings as separate types, we might be able to do some de-duplication of data in a future release (e.g. two items *Vol. 1* and *Vol. 2*, and a holdings record *Vols. 1 and 2*).
+This would be quite expensive to do, so we're not going to spend significant time on that until the One Website Project is finished.
+There are better places that time/effort can be spent.
+
+
+
+## Examples
+
+<details>
+<summary>A bib with a single physical holdings record</summary>
+
+b10032538 (<a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1003253?marcData=Y&lang=eng">Library</a>/<a href="https://api.wellcomecollection.org/catalogue/v2/works/esstapc9?include=items">API</a>)
+
+```json
+{
+  "type": "Work",
+  "id": "esstapc9",
+  "holdings": [
+    {
+      "identifiers": [
+        {
+          "identifierType": {"id": "sierra-system-number", "label": "Sierra system number", "type": "IdentifierType"},
+          "value": "1063290"
+        }
+      ]
+      "title": "Vol. 1-7",
+      "locations": [
+        {
+          "locationType": {
+            "id": "closed-stores",
+            "label": "Closed stores",
+            "type": "LocationType"
+          },
+          "label": "Closed stores",
+          "type": "Location"
+        }
+      ],
+      "type": "Holdings"
```

Comment:
As above, if `holdings` is a list, this should probably be singular, like `Item`. Have you seen multiple holdings in the data? It would be good to see an example of that to see how things split across multiple vs an enumeration within.

Reply:
See below!